### PR TITLE
vahram/vahram.developer@gmail.com tests for convert between int and char

### PR DESCRIPTION
**Pull request by vahram-papazyan**

### CODE REVIEW

File: core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestCharToInt.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package typeconversion
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import kotlin.test.assertNull
+
+class TestCharToInt {
+    @Test fun `Given a Char, When it's a valid Unicode character, Then should convert to the corresponding Int`() {
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestCharToInt.kt:9]: Let's add some newlines here

```suggestion
class TestCharToInt {

    @Test
    fun `Given a Char, When it's a valid Unicode character, Then should convert to the corresponding Int`() {
```

File: core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package typeconversion
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class TestIntToChar {
+    @Test
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt:8]:
```suggestion
class TestIntToChar {

    @Test
```

File: core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        // The escaped Unicode representation of 'A'
+        assertEquals('\u0041',  'A')
+    }
+
+    @Test
+    fun `Given a Char, When it's valid char, Then should convert to Int`() {
+        assertEquals('A'.code, 65)
+        assertEquals('\u0041'.code, 65)
+    }
+    @Test
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt:20]:
```suggestion
    }

    @Test
```

File: core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package typeconversion
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class TestIntToChar {
+    @Test
+    fun `Given an Int, When it's valid number, Then should convert Int to the corresponding Unicode character`() {
+        assertEquals('A',  'A')
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt:10]: should this actually be comparing some number with `.toChar()`?

Comment (vahram-papazyan) [reply core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt:10]: Yes, this was a typo.

File: core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestCharToInt.kt [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import kotlin.test.assertNull
+
+class TestCharToInt {
+
+    @Test
+    fun `Given a Char, When it's a valid Unicode character, Then should convert to the corresponding Int`() {
+        assertEquals('A'.code, 65)
```
[the remark this answers is not in the document]
Comment (vahram-papazyan) [reply core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestCharToInt.kt]: Done

**Pull request merged**